go2_simulation/simple_wrapper.py

```diff
@@ -30,15 +29,9 @@ def __init__(self, model, geom_model, visual_model, q0, args):
 
         # Simulation parameters
         self.simulator = simple.Simulator(model, self.data, geom_model, self.geom_data)
-        # admm
-        self.simulator.admm_constraint_solver_settings.absolute_precision = args["tol"]
-        self.simulator.admm_constraint_solver_settings.relative_precision = args["tol_rel"]
-        self.simulator.admm_constraint_solver_settings.max_iter = args["maxit"]
-        self.simulator.admm_constraint_solver_settings.mu = args["mu_prox"]
-        # pgs 
-        self.simulator.pgs_constraint_solver_settings.absolute_precision = args["tol"]
-        self.simulator.pgs_constraint_solver_settings.relative_precision = args["tol_rel"]
-        self.simulator.pgs_constraint_solver_settings.max_iter = args["maxit"]
+        self.simulator.contact_solver_info = pin.ProximalSettings(
+            args["tol"], args["tol_rel"], args["mu_prox"], args["maxit"]
+        )
```

Comment:
`contact_solver_info` does not exist anymore, please revert this change. in my [PR](https://github.com/inria-paris-robotics-lab/go2_simulation/pull/9/files) I changed these things on purpose to make it work with the latest Simple API.